src/cts/src/HTreeBuilder.cpp

```diff
@@ -33,14 +33,15 @@
 //
 ///////////////////////////////////////////////////////////////////////////////
 
+#include "HTreeBuilder.h"
```

Comment:
Why is this moved?

Reply:
It was moved after running clang-format.

Reply:
clang-format moved it from the correct order to out-of-order? That doesn't seem right...